src/CMakeLists.txt

```diff
@@ -1,10 +1,6 @@
 ############################################################################
 #    Copyright (C) 2014 by Ahmed Charles - acharles@outlook.com            #
-<<<<<<< HEAD
 #    Copyright (C) 2015-2016 by Stephen Lyons - slysven@virginmedia.com    #
-=======
-#    Copyright (C) 2015 by Stephen Lyons - slysven@virginmedia.com         #
->>>>>>> SlySven/release_30
```

Comment:
Just in case my Repo is being taken in vain, my release_30 branch is synced to the Mudlet/release_30 at this point, they are both on the same SHA1 here.